openfisca_us/tests/baseline/benefit/SNAP/amount.yaml

```diff
@@ -0,0 +1,230 @@
+- name: 'family of 4, no member(s) with disabilities, income of 24k/year (Standard Test)'
+  period: 2021
+  absolute_error_margin: 0
+  input:
+    people:
+      person_1:
+        age: 30
+      person_2:
+        age: 35
+      person_3:
+        age: 17
+      person_4:
+        age: 15
+    spm_units:
+      spm_unit:
+        snap_gross_income: 2000 * 12
+        members: [person_1, person_2, person_3, person_4]
+    households:
+      household:
+        state_group: CONTIGUOUS_US
+  output:
+    snap: (837 - 315) * 12
+
+
+- name: family of 4, member(s) with disabilities, income of 24k/year (Standard Test)
+  period: 2021
+  absolute_error_margin: 0
+  input:
+    people:
+      person_1:
+        age: 30
+      person_2:
+        age: 35
+      person_3:
+        age: 17
+      person_4:
+        age: 15
+    spm_units:
+      spm_unit:
+        snap_gross_income: 2000 * 12
+        members: [person_1, person_2, person_3, person_4]
+    households:
+      household:
+        state_group: CONTIGUOUS_US
+  output:
+    expected_food_contribution: (837 - 218.55) * 12
+ 
+
+- name: Individual, $50k income (Extreme Test)
+  period: 2019
+  absolute_error_margin: 0
+  input:
+    people:
+      person_1:
+        age: 30
+    spm_units:
+      spm_unit:
+        snap_gross_income: 4166 * 12
+        members: [person_1]
+    households:
+      household:
+        state_group: CONTIGUOUS_US
+  output:
+    snap: 0 * 12
+
+
+- name: Individual, $0 income (Extreme Test)
+  period: 2019
+  absolute_error_margin: 0
+  input:
+    people:
+      person_1:
+        age: 30
+    spm_units:
+      spm_unit:
+        snap_gross_income: 0 * 12
+        members: [person_1]
+    households:
+      household:
+        state_group: CONTIGUOUS_US
+  output:
+    snap: 204 * 12
+
+
+- name: single person, 10k income, GU (State Group Test)
+  period: 2021
+  absolute_error_margin: 0
+  input:
+    people:
+      person_1:
+        age: 30
+    spm_units:
+      spm_unit:
+        snap_gross_income: 833 * 12
+        members: [person_1]
+    households:
+      household:
+        state_group: GU
+  output:
+    snap: (300 - 0)
+
+
+- name: single person, 10k income, disabled members present, GU (State Group Test)
+  period: 2021
+  absolute_error_margin: 0
+  input:
+    people:
+      person_1:
+        age: 30
+        is_disabled_or_elderly_for_snap: True
+    spm_units:
+      spm_unit:
+        snap_gross_income: 833 * 12
+        members: [person_1]
+    households:
+      household:
+        state_group: GU
+  output:
+    snap: (300 - 0) * 12
+
+
+- name: single person, 10k income, VI (State Group Test)
+  period: 2021
+  absolute_error_margin: 0
+  input:
+    people:
+      person_1:
+        age: 30
+    spm_units:
+      spm_unit:
+        snap_gross_income: 833 * 12
+        members: [person_1]
+    households:
+      household:
+        state_group: VI
```

Comment:
Could you add VI to the `StateGroup` enum?

https://github.com/PolicyEngine/openfisca-us/blob/420e677d58449025642a433c71c44a6314f93940/openfisca_us/variables/demographic/household.py#L112-L117

@nikhilwoodruff can we test that enum values provided are part of the enum? I guess this would be an openfisca-core request?